docs/Biconomy AA Stack/Account/_category_.json

```diff
@@ -1,8 +1,8 @@
 {
-  "label": "Smart Accounts",
+  "label": "Smart Accounts Platform",
```

Comment:
This is causing the build to fail. Can ask @Rahat-ch to do this change and it requires more changes than just changing the name.

Reply:
I have changed it back for now.